### ISSUE TITLE
date range problem

### ISSUE THREAD

**Issue by ChenCong6837** [Hacker0x01/react-datepicker#1650]
data range use two DatePicker component, the range shows from the selectsStart acts as expection when focus the DataPicker, but selectsEnd don't ( the startDate props shows less 1 day )
for example : 
```javascript
const startDate = new Date()  //today is 2019.2.28, startDate: 2019.2.28
const date= new Date()
const endDate = new Date(date.setDate(date.getDate() - 6)) // endDate: 2019.2.22

<DatePicker
    selectsStart
    startDate={startDate}
    endDate={endDate}
    selected={startDate}
    onChange={this.handleChangeStart}
/>
<DatePicker
    selectsEnd
    startDate={startDate}
    endDate={endDate}
    selected={endDate}
    onChange={this.handleChangeEnd}
/>
```
### Expected behavior
when I focus selectsStart element, it shows range: ```2019.2.22~2019.2.28```
when I focus selectsEnd element, it shows range: ```2019.2.22~2019.2.28```

### Actual behavior
when I focus selectsStart element, it shows range: ```2019.2.22~2019.2.28```
when I focus selectsEnd element, it shows range: **```2019.2.23~2019.2.28```**

**Comment by lionskape** [comment]
I had same issue, did u find solution?

**Comment by lionskape** [comment]
UPD: after research I found real issue and solution.

Dates compare with time part. Actually parameter startDate for "selectsEnd" does not means "date", it means "date time".

Problem are inside "src/day.jsx:81" (isInSelectingRange) function. I do not go deeper.
For tmp solution I reseted time part of startDate parameter to zero and it starts work.

I do not know how it should work, but I think that real issue now is difference in compare of "selectsStart" and "selectsEnd" parts of range.
